flake.nix

```diff
@@ -0,0 +1,46 @@
+{
+  description = "Dev shell for charts";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = {
+    self,
+    nixpkgs,
+    flake-utils,
+  }:
+    flake-utils.lib.eachDefaultSystem (system: let
+      pkgs = import nixpkgs {inherit system;};
+      devEnvVars = {
+        KUBE_CONFIG_PATH = "~/.kube/config";
+        KUBE_CTX = "k3d-k3s-default";
+      };
+    in
+      with pkgs; {
+        devShells.default = mkShell (devEnvVars
+          // {
```

Comment:
Do you know why we need the double slash here?

Reply:
It is basically a concat operation, using the `//` syntax we are merging the 2nd set to the `devEnvVars` set. A more detailed explanation can be found here https://nixos.org/manual/nixpkgs/stable/#function-library-lib.attrsets.mergeAttrsList